src/CommonLib/LDAPUtils.cs

```diff
@@ -1428,6 +1551,55 @@ internal string ResolveDomainNetbiosToDns(string domainName)
 
             return domainName.ToUpper();
         }
+        
+        /// <summary>
+        /// Gets the range retrieval limit for a domain
+        /// </summary>
+        /// <param name="domainName"></param>
+        /// <returns></returns>
+        public int GetDomainRangeSize(string domainName = null)
```

Comment:
Consider changing this to `GetDomainRangeSize(string domainName = null, int defaultSize = 750)`.

Reply:
Could also use an associated test if you've got the bandwidth